core/templates/dev/head/domain/utilities/UrlInterpolationService.js

```diff
@@ -79,8 +79,8 @@ oppia.factory('UrlInterpolationService', [
         var INVALID_VARIABLE_REGEX = /(<{2,})(\w*)(>{2,})/;
 
         // Parameter values can only contain alphanumerical characters, spaces,
-        // hyphens, underscores, or periods.
-        var VALID_URL_PARAMETER_VALUE_REGEX = /^(\w| |_|-|[.])+$/;
+        // hyphens, underscores, periods or the equal to symbol.
+        var VALID_URL_PARAMETER_VALUE_REGEX = /^(\w| |_|-|[.]|=)+$/;
```

Comment:
Oh, no, I think this won't work. According to RFC 3986 the '=' character is reserved. You will need to escape it before using the UrlInterpolationService, and you can't modify this. Make sure to test end-to-end to ensure that everything still works, even with '=' symbols.

That said, I'm slightly confused. We are actually generating feedback thread IDs with bad chars in them? Looking at the code, I thought we were generating them only from a base64 charset, which should not include the '=' sign...